packages/perseus/src/widgets/__tests__/numeric-input.test.ts

```diff
@@ -212,6 +213,40 @@ describe("static function validate", () => {
         `);
     });
 
+    // Don't default to validating the answer as a pi answer
+    // if answerForm isn't set on the answer.
+    // The answer value, userInput.currentValue, and
+    // the omission of answerForms in the answer are
+    // important to the test.
+    // https://khanacademy.atlassian.net/browse/LC-691
+    it("doesn't default to validating pi", () => {
+        const rubric: Rubric = {
+            answers: [
+                {
+                    maxError: null,
+                    message: "",
+                    simplify: "required",
+                    status: "correct",
+                    strict: true,
+                    value: 45.289,
+                },
+            ],
+            labelText: "",
+            size: "normal",
+            static: false,
+            coefficient: false,
+        };
+
+        const useInput = {
+            currentValue: "45.282",
```

Comment:
I noticed in the PR description that you talk about the answer being close to some factor if `pi/12`.  

Is this, or the value on line 231 some factor of pi?  If so, could you note that in comments here (saves time instead of having to look up the bug in Jira)?